doc/api/util.md

```diff
@@ -2245,3 +2246,4 @@ Deprecated predecessor of `console.log`.
 [list of deprecated APIS]: deprecations.html#deprecations_list_of_deprecated_apis
 [semantically incompatible]: https://github.com/nodejs/node/issues/4179
 [util.inspect.custom]: #util_util_inspect_custom
+[object spread notation]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax
```

Comment:
Nit: this should go before `[semantically incompatible]:`.

Reply:
I would love to have a linting rule for that :)